### ISSUE TITLE
Where to tap TX9 on FT991 non-A?

### ISSUE THREAD

**Issue by ianfromnyc** [rfrht/FT991A-PAT#16]
Looking at https://github.com/rfrht/FT991A-PAT/wiki/installation#tx9 and comparing the image to my 991 non-A, it appears that the highlighted section is not present on my radio.  Might it be possible to point me toward an alternate location to tap the TX9?

Here's an photo from the same section on my radio, in case I'm just not seeing it.
![IMG_1856](https://user-images.githubusercontent.com/1969818/113484261-48619e00-949f-11eb-8327-2b9701aa7aba.jpeg)

Thanks

**Comment by rfrht** [comment]
Hello there, Ian!

First of all - **thank you very much** for reporting this issue.

You are right, this component is non-existent in FT-991.

**However**, while finding for a suitable place I found an even better place to tap the signal from - it is the collector (the lid) of Q1061. Larger component, easier to solder and bonus point - common to both FT-991 and FT-991A.

Schematic:
![Screenshot from 2021-04-03 15-40-45](https://user-images.githubusercontent.com/3044503/113488636-ad17fb00-9495-11eb-98a4-f4fb8a46211a.png)

Board:
![Screenshot from 2021-04-03 15-39-51](https://user-images.githubusercontent.com/3044503/113488646-ba34ea00-9495-11eb-83c8-134ec4158fa9.png)

Can I ask you a favour? Can you please take a picture of this point so I can use your picture in the Wiki?

Your previous picture looks gorgeous and I'd appreciate a lot if you could provide the picture of the actual board - and give you the credits of course. 

Thank you very much for reporting!

**Comment by ianfromnyc** [comment]
Hi Rodrigo,

Thank you for replying so quickly!  Here are the photos you asked for.

Q1061 is underneath what I believe is the PLL module and a RF shield.
  
First, remove the 4 screws at the corners of the shield:

![IMG_1857](https://user-images.githubusercontent.com/1969818/113494110-5a623180-94dd-11eb-8eb2-28396387ffc3.jpeg)

Next, remove the PLL module.  It is only attached by the pin headers on two sides:

![IMG_1858](https://user-images.githubusercontent.com/1969818/113494136-7665d300-94dd-11eb-946f-25105fbe364f.jpeg)

Now you can access Q1061:

![IMG_1859](https://user-images.githubusercontent.com/1969818/113494155-9bf2dc80-94dd-11eb-90c7-f5e27868fcf4.jpeg)

One more question: it looks like I'll need a standoff to mount the panadapter using the mainboard mounting screw.  Do you know what size the screw is?  Alternatively, is there another way to ground the panadapter without screwing it down?

Thanks

Ian P
M7WMT

**Comment by rfrht** [comment]
Oooh no no no **stop right now Ian**

**Comment by rfrht** [comment]
This is bad news all way long man, i didn't know it was underneath the PLL Unit, damn! Don't tap from there!

There are two resistors in this side of the board (the other TX9 line elements are underneath the Main unit and we don't want that)

Give me a few minutes more and and I'll provide you the two other options - then you can find out what's the better option

And will reply the other questions too. Stand by!

**Comment by ianfromnyc** [comment]
Okay, I've already soldered the wire to that transistor, but I can move it somewhere else.  Out of curiosity, what makes that such a bad spot?

Regarding the mounting screw, I had a look at the upstream repo and it seems OE2DOR soldered a M3 standoff to the screw then mounted the panadapter to that.  I'll try that unless you have a better suggestion.

**Comment by rfrht** [comment]
Ok Ian

Besides Q1061 we have the following components in TX9 line, namely:

* R1429
* R1601
* R1625
* R1754
* R1755
* R1759

TL;DR: Only R1601 and R1625 are suitable, because they are on the upper side of the board. All other compoonents are underneath. 

R1429:
![Screenshot from 2021-04-03 21-24-18](https://user-images.githubusercontent.com/3044503/113494980-26c6dd80-94c4-11eb-93ee-7a3979f6f365.png)

R1601 and R1625:
![Screenshot from 2021-04-03 21-24-44](https://user-images.githubusercontent.com/3044503/113494981-27f80a80-94c4-11eb-8a26-ce4179bb8ba3.png)

* R1754, R1755 and R1759:
![Screenshot from 2021-04-03 21-25-01](https://user-images.githubusercontent.com/3044503/113494983-27f80a80-94c4-11eb-89f1-6e5cbb613cd3.png)


Now let me get the location for 1601 and 1625.

**Comment by rfrht** [comment]
> Okay, I've already soldered the wire to that transistor, but I can move it somewhere else. Out of curiosity, what makes that such a bad spot?
Man, I am sorry - I did not know really that there was a board above that spot, Ian. 

The reason this is a bad spot, well the problem number zero is the shield - The shield presence means it has to be a very controlled in terms of noise.

The wire traveling from the SDR can become an unintentional radiator *right below* the PLL Unit risking wreak havoc in your frequency stability - and we don't want that.

> Regarding the mounting screw, I had a look at the upstream repo and it seems OE2DOR soldered a M3 standoff to the screw then mounted the panadapter to that. I'll try that unless you have a better suggestion.

In my case, I worked with what I had, Ian - and that was not exactly pretty.

I used those VGA nuts and 'forced' it a bit on the screw slot. I didn't have anything that would be that tall and would be a perfect fit :-(

Thus, the VGA nut.

**Comment by ianfromnyc** [comment]
Thanks for the warning, I'm thinking R1625 is the better choice since it's closer.  Though, it is right next to (but still outside the shield of) the PLL unit, so maybe not?

What do you think?

**Comment by rfrht** [comment]
So here is the PCB location, Ian.

R1625
![Screenshot from 2021-04-03 21-42-00](https://user-images.githubusercontent.com/3044503/113495254-a8b80600-94c6-11eb-9f09-2d490e918d47.png)

R1601
![Screenshot from 2021-04-03 21-41-09](https://user-images.githubusercontent.com/3044503/113495252-a6ee4280-94c6-11eb-875c-8ce383a69b07.png)

**Fun fact**: Both are common to 991A too :-P

I think that the R1601 would be a better idea, in a less cluttered part of the PCB board - and at that point it controls the TX GND signal to Linear Amplifier, *far far away* from R1625, which is nearby the critical signal path.

Now it is your call - I don't know what are the obstacles or challenges for 1601.

Let me know your conclusion?

And apologies again for your excursion underneath PLL unit, ugh.

**Comment by ianfromnyc** [comment]
Thank you very much for the advice.  I'm a novice so I really appreciate it.

I'll try the R1601 and let you know how it goes.  I'll also make sure to get some nice pictures for the Wiki.  I may not get to it until Tuesday because of the holiday, though.

**Comment by rfrht** [comment]
Take your time - and I wish you and your family a happy Easter!

Thanks Ian, that was fun (other than still baffled about the PLL Unit)

**Comment by ianfromnyc** [comment]
Hi Rodrigo,

I got the board installed and the panadapter appears to be working, thanks again for all the hard work!  Here's a photo of the TX9 wire soldered on to R1601:

![IMG_1861](https://user-images.githubusercontent.com/1969818/114060150-fc429f00-988c-11eb-97a4-3837ddc4e8e7.jpeg)


Unfortunately, I must have damaged the FT-991 during the installation because now the radio is deaf on HF.  144 and 430 still work.  I think it might be something to do with how I soldered the IF output to Q1088 or Q1102, which I had to try a few times before I got a good connection.  If I had damaged either of those switches, would it deafen the radio on HF but not V/U?  In any case, this is totally my fault.

I wish I had better news but I do appreciate all the work that went into this project and all your guidance on finding the TX9.

Thanks again

Ian P

**Comment by rfrht** [comment]
Oooooh boy Ian man I'm so sorry to hear man

Did you reconnect firmly the coax that you removed to solder the TX9?

>  If I had damaged either of those switches, would it deafen the radio on HF but not V/U?

Okay look at the below annotated schematic:

![Screenshot from 2021-04-08 13-54-55](https://user-images.githubusercontent.com/3044503/114067518-49b21280-9873-11eb-9fd1-79906530fa32.png)

If a switch is damaged, both presents different symptoms:

* If Q1102: Q1102 selects the roofing filter, and if you get into defect over there, you would get signal problems when switching between SSB, that uses 3K filter and FM, that uses the 16K filter.
* If Q1088: No HF or V/U signal, irrespective of the mode.

The good thing is these switches are readily available, the sad part is DigiKey has no stock

https://www.digikey.com/en/products/detail/psemi/4259-63/2614473

However, if you built your panadapter, smile: It is the very same RF switch that I used in the board design.

**Comment by ianfromnyc** [comment]
Thanks again for the very detailed analysis. I've ordered the switches anyway, just in case I decide to try replacing them.  I have a few weeks of backorder lead time during which to think it over.

**Comment by rfrht** [comment]
Feel free to get in touch here or by e-mail. QRZ me and email is over there.
